src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -104,6 +104,10 @@ adapters.sovrn.enabled=true
 adapters.sovrn.endpoint=http://localhost:8090/sovrn-exchange
 adapters.sovrn.pbs-enforces-gdpr=true
 adapters.sovrn.usersync.url=//sovrn-usersync
+adapters.sharethrough.enabled=true
+adapters.sharethrough.endpoint=http://localhost:8090/sharethrough
+adapters.sharethrough.pbs-enforces-gdpr=true
+adapters.sharethrough.usersync.url=//sharethrough-usersync
```

Comment:
You've added new bidder to test-application properties, but forgot to make a separate Integration test for it.

Please add an integration test (check Adform for example).